### DISCUSSION TITLE
Deploy Application without git

### DISCUSSION THREAD

**Discussion by iamdineshkumar** [coollabsio/coolify#457] [Feature Requests]
I understand at present application can be deployed through repository scripts thats wonderful. But if there is an option to deploy without git it will be convenient in certain way. For an example we can deploy build PWA directly without repository(in my case i am not able to deploy Flutter based PWA).

**Comment by vasani-arpit** [comment]
Git(version control) is recommended in any coding project. 

But still if you want to deploy an app directly from your local machine then I suggest to wait for next major version. I think @andrasbacsai is working on adding CLI tool to deploy apps directly from local machine as per #58

**Reply by iamdineshkumar** [reply]
I am completely agree with it. As I stated above, I am having git for flutter application but when i need to deploy as web application currently there is no docker environment to support it. But I can locally generate pwa and then compress it to deploy.
